_overviews/scala3-migration/incompat-syntactic.md

```diff
@@ -237,3 +238,13 @@ The solution is to choose another valid identifier, for instance `T`.
 
 However, `+` and `-` still are valid type identifiers in general.
 You can write `type +`.
+
+## Unicode escapes
+
+Unicode escapes of the form `\u0061` are no longer natively supported in any position by the compiler.
```

Comment:
"natively supported" seems unclear to me. not sure what best replacement is, maybe:

```suggestion
Unicode escapes of the form `\u0061` are no longer allowed in arbitrary locations in source code.
```